os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -136,7 +139,19 @@ u8 *_tizenrt_malloc(u32 sz)
 
 u8 *_tizenrt_zmalloc(u32 sz)
 {
-	return kmm_zalloc(sz);
+	void *pbuf = NULL;
```

Comment:
Doesn't this change all the existing code where _tizenrt_zmalloc is used? I think aligned malloc takes up more space. So, may be adding a new function might be good. Please consider this suggestion, its not a mandatory change.

Reply:
This change should be a different commit from spi dma change. Please split commits for them.

Reply:
updated with new commit